assets/_scss/components/_search.scss

```diff
@@ -0,0 +1,102 @@
+$usa-btn-small-width:   4.5rem;
+$usa-btn-medium-width:  8.5rem;
+$usa-btn-big-width:     11.6rem;
+
+@mixin usa-search-submit-small {
+  background-image: url('../img/search.png');  
+  background-image: url('../img/search.svg');  
+  background-position: 50%;
+  background-repeat: no-repeat;
+  width: $usa-btn-small-width;
+}
+
+@mixin usa-search-submit-default {
+  @include media($mobile-up) {
+    background-image: none;
+    width: $usa-btn-medium-width;
+  }
+
+  .usa-search-submit-text {
+    display: none;
+
+    @include media($mobile-up) {
+      display: block;
+    }
+  }
+}
+
+input[type="search"] {
+  border: {
+    bottom-right-radius: 0;
+    right: none;    
+    top-right-radius: 0;    
+  }
+  float: left;
+  height: 3.3rem;
+  width: calc(100% - #{$usa-btn-small-width});
+}
+
+.usa-search {
+  max-width: none;
+}
+
+.usa-search_bar {
```

Comment:
This should actually be `usa-search-bar` because the bar in this case is a sub element of the search. Is that the correct thinking?